charts/histogram/src/index.js

```diff
@@ -34,6 +35,11 @@ export default function supernova(env) {
       data: {
         targets: [dataDefinition],
       },
+      exportProperties: ({ propertyTree, hypercubePath, viewDataMode }) =>
```

Comment:
Don't we want to add corresponding change into `exportTableProperties` for the histogram as well, as you did for the boxplot:

```
return exportProperties({ propertyTree, viewDataMode: true });
```

Just want to double check so you haven't forgotten about it?

Reply:
Done

Reply:
I don't think it will be used at the moment, but I think this makes it possible to remove the exportTableProperties from the client (if no other extensions are using it)